<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Citra</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>src/pcafe/res/icon3_64x64.ico</normaloff>src/pcafe/res/icon3_64x64.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_File"/>
    <addaction name="action_Load_Symbol_Map"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Emulation">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_Pause"/>
    <addaction name="action_Stop"/>
    <addaction name="separator"/>
    <addaction name="action_Configure"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Single_Window_Mode"/>
    <addaction name="actionDisplay_widget_title_bars"/>
    <addaction name="action_Hotkeys"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Emulation"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_File">
   <property name="text">
    <string>Load File...</string>
   </property>
  </action>
  <action name="action_Load_Symbol_Map">
   <property name="text">
    <string>Load Symbol Map...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About Citra</string>
   </property>
  </action>
  <action name="action_Single_Window_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Single Window Mode</string>
   </property>
  </action>
  <action name="action_Hotkeys">
   <property name="text">
    <string>Configure &amp;Hotkeys ...</string>
   </property>
  </action>
  <action name="action_Configure">
   <property name="text">
    <string>Configure ...</string>
   </property>
  </action>
  <action name="actionDisplay_widget_title_bars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Dock Widget Headers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Configure</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnConfigure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisplay_widget_title_bars</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>OnDisplayTitleBars(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnConfigure()</slot>
  <slot>OnDisplayTitleBars(bool)</slot>
 </slots>
</ui>
